### PR TITLE
Added E Commerce website

### PR DESCRIPTION
**Pull request by rahulch-1**
This is an E-Commerce Website made using HTML, CSS, JavaScript which has products with their description and their reviews.

### CODE REVIEW

**Comment by iamwatchdogs** [comment]
@rahulch-1 Please make the required changes to pass the lining checks, so that we can review and merge your PR. And don't worry about the files that end with `**.min.js` as the eslinter configs have been updated to ignore those files.

File: E-commerce website/.github/ISSUE_TEMPLATE/bug.yaml
Comment (iamwatchdogs) [inline E-commerce website/.github/ISSUE_TEMPLATE/bug.yaml:1]: - [ ] Please remove the `.github`  directory and all the files within it.

File: E-commerce website/css/.DS_Store
Comment (iamwatchdogs) [inline E-commerce website/css/.DS_Store:1]: - [ ]  Remove all `.DS_Store` file present anywhere within your project.

File: E-commerce website/js/.DS_Store
Comment (iamwatchdogs) [inline E-commerce website/js/.DS_Store:1]: - [ ] Remove all `.DS_Store` files present within your project.

File: E-commerce website/js/about.js
Comment (iamwatchdogs) [inline E-commerce website/js/about.js:1]: - [ ] Fix your code to pass the lining checks *(about.js)*.

File: E-commerce website/js/revolution/.DS_Store
Comment (iamwatchdogs) [inline E-commerce website/js/revolution/.DS_Store:1]: - [ ] Remove all the `.DS_Store` files present within your projects.

File: E-commerce website/js/revolution/js/.DS_Store
Comment (iamwatchdogs) [inline E-commerce website/js/revolution/js/.DS_Store:1]: - [ ] Remove all `.DS_Store` files within your projects.

**Review by iamwatchdogs — CHANGES_REQUESTED** [review]
Please make the requested changes

File: E-commerce website/Readme.md
Comment (iamwatchdogs) [inline E-commerce website/Readme.md:1]: (Optional) Can you add a small description explaining your project to others?

File: E-commerce website/js/revolution/css/openhand.html
Comment (iamwatchdogs) [inline E-commerce website/js/revolution/css/openhand.html:1]: > [!IMPORTANT]
> If this file is empty and does nothing, make sure to remove it.

File: E-commerce website/js/revolution/css/closedhand.html
Comment (iamwatchdogs) [inline E-commerce website/js/revolution/css/closedhand.html:1]: > [!IMPORTANT]
> If this file is empty and does nothing, make sure to remove it.

**Comment by iamwatchdogs** [comment]
@rahulch-1 don't worry about the lining checks for now, as the only issues that are reported for creating unused variables _(which were used in other files)_ and not recognising the jquery syntax.

**Review by iamwatchdogs — CHANGES_REQUESTED** [review]
@rahulch-1 The only changes that were made to resolve the code review issue is to remove a single `.DS_Store` file from your projects. You still need to make the following changes:

- [ ] Remove the directory/file names `.github`.
- [ ] Remove all `.DS_Store` files.
- [ ] Reply to the previously mentioned comments about `openhand.html` and `close hand.html` _(You can find them in our PR conversation)_

> [!NOTE]
> Don't worry about the linting checks as we're ignoring them for your case. Just make the above mentioned changes.

File: E-commerce website/js/revolution/css/closedhand.html
Comment (rahulch-1) [reply E-commerce website/js/revolution/css/closedhand.html:1]: Done @iamwatchdogs I removed those unused html files

**Comment by rahulch-1** [comment]
Now I removed the .github directory and all the .DS_Store files also

**Comment by rahulch-1** [comment]
Thanks for helping me out

**Comment by iamwatchdogs** [comment]
@rahulch-1 you have indeed removed `.github` directory but you haven't removed all `.DS_Store` file.

**Comment by iamwatchdogs** [comment]
All of these remaining `.DS_Store` file paths are mentioned in previous code review that are present in this conversation.

**Comment by rahulch-1** [comment]
Sorry, but I am not able to find any .DS_store files now

**Comment by iamwatchdogs** [comment]
> Thanks for helping me out

It's okay @rahulch-1. Just so you know, this linter makes sure each and every file maintains its integrity and works as intended without any syntactical errors. In some cases, the linter that we have used checks purely for JavaScript and doesn't recognise any external framework. This is the reason why people set up linter within their projects that are built using frameworks, as these linter will be configured according to the framework avoiding any linting issues. And the way we configured our linter for this repo is for pure javascript files and not medium to big projects built using frameworks like React. 

We respect your patience and willing to contribute, but I, @iamwatchdogs, as the main maintainer have to make sure you give the best from your side while maintaining the whole integrity of the repo. So, please have some patience as when we request to make a few changes. Hope this helps you understand what we at @Grow-with-Open-Source are trying to do.

**Comment by iamwatchdogs** [comment]
> Sorry, but I am not able to find any .DS_store files now

Sorry for that @rahulch-1, Yes all the files were removed.

**Review by iamwatchdogs — APPROVED** [review]
Request Changes were made. Reviewed and Approved by @iamwatchdogs.

**Comment by rahulch-1** [comment]
yes, @iamwatchdogs I completely understand the need for a specialized linter setup

**Comment by iamwatchdogs** [comment]
Thank you for contributing, @rahulch-1. Please check your contribution on [GitHub Pages](https://grow-with-open-source.github.io/Javascript-Projects/).

**Pull request merged**